lib/state/locale.dart

```diff
@@ -23,4 +23,8 @@ extension LocaleExtenstion on AppLocale {
   Locale get locale => _langCodes[this];
 
   String get tag => _langTags[this];
+}
+
+extension StringLocaleExtension on String {
+  AppLocale toAppLocale() => AppLocale.values.firstWhere((element) => element.tag == this);
```

Comment:
Didn't understand, why is this extension on String?

Reply:
Moved into appLocale extensions